modules/service/src/main/scala/lucuma/odb/graphql/enums/Enums.scala

```diff
@@ -86,10 +87,33 @@ final class Enums(
 
   }
 
+  enum ProposalStatus(val tag: String, ordinal: Short) {
+    private val meta = enumMeta.proposalStatus.getOrElse((tag, ordinal), sys.error(s"t_proposal_status missing tag `$tag`, ordinal `$ordinal`"))
+
+    def name: String = meta.name
+
+    // Note: The order here controls the Order of the Enumerated in the ODB, so keep them in order. 
```

Comment:
All the existing dynamic enums get ordered by the `tag` in the API. I wanted a consistent, non-tag order between API and clients so I added the ordinal. Couldn't think of a better way.

Reply:
You could prepend a number to the tag since the tags aren't supposed to be for human consumption, but I guess we have kind of violated that idea already by exposing the tag as the enum value :-\

So this seems ok.